sw/airborne/autopilot.c

```diff
@@ -160,6 +161,13 @@ void autopilot_periodic(void)
     // after takeoff for fixedwing
     // when in flight for other firmwares
     RunOnceEvery(PERIODIC_FREQUENCY, autopilot.flight_time++);
+
+    if(!fast_reboot) {
```

Comment:
Why not in the `autopilot_init` function ? It seems to be called only once.

Reply:
Needs to be called only when `in_flight` else you will always skip the bootloader at a reboot. That makes it very hard to flash the autopilot. But will find a correct place